main.py

```diff
@@ -0,0 +1,32 @@
+from app.animal import Animal, Carnivore, Herbivore
+
+
+if __name__ == "__main__":
+    lion = Carnivore("Simba")
+    print(len(Animal.alive) == 1)
+    print(isinstance(Animal.alive[0], Carnivore) is True)
+
+    rabbit = Herbivore("Susan")
+    rabbit.hide()
+    print(rabbit.hidden is True)
```

Comment:
Code Efficiency: Here also, you can use the `assert` statement. For example, `assert rabbit.hidden`.